pages/event-resources.jsx

```diff
@@ -0,0 +1,407 @@
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyWallpaper-1-640x1136.jpg">640px x 1136px</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyWallpaper-1-768x1280.jpg">768px x 1280px</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyWallpaper-1-1920x1200.jpg">1920px x 1200px</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyWallpaper-1-2560x1440.jpg">2560px x 1440px</LogoAssetLink>
+          </LogoAsset>
+          <LogoAsset head="Certificate"
+          alt="Certificate Image"
+          src1x="/img/event-resources-page/resource-thumbnails-08.png"
+          src2x="/img/event-resources-page/resource-thumbnails-08@2x.png">
+            <RemixLink href="https://makerparty2014.makes.org/thimble/LTEzNDM3NTAxNDQ=/maker-party-love-bomb">Download EPS</RemixLink>
+          </LogoAsset>
+          <LogoAsset head="Table Cloth Design"
+          alt="Table Cloth Design Image"
+          src1x="/img/event-resources-page/resource-thumbnails-04.png"
+          src2x="/img/event-resources-page/resource-thumbnails-04@2x.png">
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyTableClothDesign.eps">Download EPS</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyTableClothDesign.png">Download PNG</LogoAssetLink>
+          </LogoAsset>
+          <LogoAsset head="Buttons"
+          alt="Buttons Image"
+          src1x="/img/event-resources-page/resource-thumbnails-06.png"
+          src2x="/img/event-resources-page/resource-thumbnails-06@2x.png">
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyButtons.eps">Download EPS</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/MakerPartyButtons.png">Download PNG</LogoAssetLink>
+          </LogoAsset>
+          <LogoAsset head="KUMI Papertoy"
+          alt="KUMI Papertoy Image"
+          src1x="/img/event-resources-page/resource-thumbnails-09.png"
+          src2x="/img/event-resources-page/resource-thumbnails-09@2x.png">
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/KUMI-Papertoy-yellow.pdf">Yellow T-sirt</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/KUMI-Papertoy-blue.pdf">Teal T-shirt</LogoAssetLink>
+            <LogoAssetLink href="https://party.webmaker.org/party-resources/KUMI-Papertoy-blank-tees.pdf">Design Your Own T-shirt</LogoAssetLink>
+          </LogoAsset>
+        </div>
+
+        <h2 id="event-details">Event Details</h2>
+        <p>There are many details to consider when planning your event. Here's a list of what you need to know:</p>
+
+        <Tabulator tabs={[
+          {
+            "head": "the basics",
+            "content": (
+              <div>
```

Comment:
Yeah, totally! I like that way too, but I also have no idea how to generate the tabulator from it, since `this.props.children` is so doggone opaque. React does have a [`React.Children`](https://facebook.github.io/react/docs/top-level-api.html#react.children) that might be useful for such a thing though.